src/app/attendees/attendees.component.html

```diff
@@ -24,7 +24,7 @@
       <a *ngFor="let person of winners" class="col-1-8" >
         <div class="module hero">
           <h4>{{person.member.name}}</h4>
-          <img src="{{this.realUrl(person.member.photo)}}">
+          <img class="winnerPhoto" src="{{this.realUrl(person.member.photo)}}">
```

Comment:
We have no such domain objects as loosers :) Every selected attendee is winner, but he/she could reject. Does it make them loosers? :)